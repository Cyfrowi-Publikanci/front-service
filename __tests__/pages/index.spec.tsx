import { shallow } from 'enzyme';

import Dashboard from '../../pages';

jest.mock('react-i18next', () => ({ useTranslation: () => ({ t: (key: string) => key }) }));

describe('Dashboard', () => {
  it('should match snapshot', () => {
    const wrapper = shallow(<Dashboard />);

    expect(wrapper).toMatchSnapshot();
  });
});
