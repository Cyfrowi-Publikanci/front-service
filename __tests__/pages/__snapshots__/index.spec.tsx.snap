// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard should match snapshot 1`] = `
<Layout>
  <Head>
    <title>
      Dashboard
    </title>
  </Head>
  <h1>
    Dashboard
  </h1>
</Layout>
`;
